packages/ui/src/components/ProfileCarousel/ProfileCarousel.tsx

```diff
@@ -13,21 +15,43 @@ export const ProfilesCarousel: React.FC<Props> = ({
   profileCardData,
   title = 'Clients',
 }) => {
-  return (
-    <div className="profile-carousel px-4 lg:px-8">
-      <p className="font-semibold text-lg py-8 leading-5">{title}</p>
+  const [searchQuery, setSearchQuery] = useState<string>('')
+  const normalizeText = (text: string) => text.replace(/\s+/g, '').toLowerCase()
+  const filteredProfileData = profileCardData.filter((profile) =>
+    searchQuery === ''
+      ? true
+      : normalizeText(profile.name) === normalizeText(searchQuery)
+  )
 
-      <div className="overflow-x-auto">
-        <Swiper spaceBetween={20} slidesPerView="auto" direction="horizontal">
-          {profileCardData.map((profile, index) => {
-            return (
-              <SwiperSlide key={index} className="!w-fit">
+  return (
+    <>
+      <div className="ml-8 mr-8 w-[fit]">
+        <div className="flex flex-row items-center justify-between">
+          <p className="font-semibold text-lg py-8 leading-5">{title}</p>
+          <p className="mr-2 text-blue-600">View all</p>
+        </div>
+        <div className="flex items-center p-2 bg-white-smoke rounded-md overflow-hidden mb-4">
+          <Icon iconName="search" />
+          <input
+            type="text"
+            value={searchQuery}
+            onChange={(e) => setSearchQuery(e.target.value)}
+            placeholder="Search by name"
+            className="bg-white-smoke rounded-2xl px-3 py-1 w-full mr-2 focus:outline-none"
+          />
+        </div>
+      </div>
+      <div className="profile-carousel px-4 lg:px-8">
+        <div className="overflow-x-auto">
+          <Swiper spaceBetween={20} slidesPerView="auto" direction="horizontal">
+            {filteredProfileData.map((profile, index) => (
```

Comment:
I didn't find the Add new client profile card in the carousal. please make it done.

Reply:
The Add New card doesn't redirect to Add customer page.